### PR TITLE
sollya 6.0 (new formula)

### PR DESCRIPTION
**Pull request by kbinani**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### CODE REVIEW

File: sollya.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Sollya < Formula
+  desc "Sollya: an environment for the development of numerical codes"
+  homepage "http://sollya.gforge.inria.fr/"
+  url "https://gforge.inria.fr/frs/download.php/file/36271/sollya-6.0.tar.bz2"
+  sha256 "05321581ba47f5e0804b712157dfc5343a268ca6c1983dce48fdd1e91d5b0a1f"
```
Comment (jonchang) [inline sollya.rb]: Please add a `head` URL for the git repository. Also consider adding a `doi` tag for the citation (like in [raxml.rb](https://github.com/Homebrew/homebrew-science/blob/master/raxml.rb)).

File: sollya.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Sollya < Formula
+  desc "Sollya: an environment for the development of numerical codes"
```
Comment (jonchang) [inline sollya.rb]: There's no need to include the software's name in the description.

**Comment by kbinani** [comment]
@jonchang Thank you for the comments! I have just pushed revised version.

**Comment by jonchang** [comment]
I've just merged this. Thanks @kbinani for contributing to Homebrew!

**Pull request closed without merging**